src/care/PluginData.h

```diff
@@ -0,0 +1,65 @@
+#ifndef _CARE_PluginData_H_
+#define _CARE_PluginData_H_
+
+#include "chai/ExecutionSpaces.hpp"
+#include "care/config.h"
+#include "care/CHAICallback.h"
+#include <vector>
+
+namespace chai {
+   struct PointerRecord;
+}
+
+namespace care{
+
+	class PluginData	{
+		public:
+			PluginData();
```

Comment:
`PluginData() = default;`

That way you can remove the definition in the .cpp file.